gdpr/impl_test.go

```diff
@@ -183,3 +183,10 @@ func assertBoolsEqual(t *testing.T, expected bool, actual bool) {
 		t.Errorf("Expected %t, got %t", expected, actual)
 	}
 }
+
+func assertStringsEqual(t *testing.T, expected string, actual string) {
```

Comment:
Is this function being used? This is the only change in the file, and it is a pure addition, so `assertStringsEqual()` couldn't have been there before.